pubnub/test/unit/dx/push_test.dart

```diff
@@ -189,6 +189,33 @@ void main() {
               #environment: null
             }));
       });
+
+      test('listPushChannels delegate supported arguments', () async {
+        fakePubnub.returnWhen(
+            #listPushChannels,
+            Future.value(
```

Comment:
You can also use [trailing commas](https://docs.flutter.dev/tools/formatting#using-trailing-commas) here, I leave it up to you